en/docs/_ci/deb-specific-version.md

```diff
@@ -0,0 +1,4 @@
+It is recommended that you lock in a specific version of Yarn, so that all your builds use the same version of Yarn, and you can test new Yarn releases before switching over. You can do this by adding the version number to the `apt-get install` call:
+```
+sudo apt-get install -y -qq yarn={{site.latest_version}}-1
```

Comment:
All our Debian packages have `-1` at the end of the version (eg. `0.16.0-1`). The idea is that it's bumped (eg. to `0.16.0-2`) if you just make changes to the .deb file without modifying the software, like if you need to rebuild it for some reason.